app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -176,14 +176,31 @@ public MediaSourceInfo getCurrentMediaSource() {
         }
     }
 
-    public StreamInfo getCurrentStreamInfo() { return mCurrentStreamInfo; }
-    public boolean canSeek() {return !isLiveTv;}
-    public boolean isLiveTv() { return isLiveTv; }
-    public int getSubtitleStreamIndex() {return (mCurrentOptions != null && mCurrentOptions.getSubtitleStreamIndex() != null) ? mCurrentOptions.getSubtitleStreamIndex() : -1; }
-    public @Nullable Integer getAudioStreamIndex() {
+    public StreamInfo getCurrentStreamInfo() {
+        return mCurrentStreamInfo;
+    }
+
+    public boolean canSeek() {
+        return !isLiveTv;
+    }
+
+    public boolean isLiveTv() {
+        return isLiveTv;
+    }
+
+    public int getSubtitleStreamIndex() {
+        return (mCurrentOptions != null && mCurrentOptions.getSubtitleStreamIndex() != null) ? mCurrentOptions.getSubtitleStreamIndex() : -1;
+    }
+
+    public @Nullable
+    Integer getAudioStreamIndex() {
```

Comment:
I don't think this should be multiline?

```suggestion
    public @Nullable Integer getAudioStreamIndex() {
```

Reply:
The formatter will split it back into multi line
Would the following be better?
```
    @Nullable
    public Integer getAudioStreamIndex() {
```

Reply:
Yeah that would be better